ui/flow.reel/flow.js

```diff
@@ -1615,34 +1611,34 @@ var Flow = exports.Flow = Component.specialize( /** @lends Flow.prototype # */ {
             }
             if (this.splinePaths.length) {
                 for (i = 0; i < length; i++) {
-                    offset = this.offset(visibleIndexes[i]);
+                    offset = this.offset(this._visibleIndexes[i]);
                     pathIndex = offset.pathIndex;
                     slideTime = offset.slideTime;
                     indexTime = this._splinePaths[pathIndex]._convertSplineTimeToBezierIndexTime(slideTime);
                     iteration = this._repetition._drawnIterations[i];
                     element = iteration.cachedFirstElement || iteration.firstElement;
                     if (indexTime !== null) {
-                        if (element.children[0]) {
+                        if (elementChildren = element.children[0]) {
```

Comment:
elementChildren is cached but it's only used for elementChildren.classList, why not cache elementChildrenClassList?